media-gfx/exiv2/files/exiv2-0.25-fix-docs.patch

```diff
@@ -0,0 +1,71 @@
+--- a/CMakeLists.txt	2015-07-19 16:04:35.363628707 +0200
++++ b/CMakeLists.txt	2015-07-19 16:28:31.051680136 +0200
+@@ -216,9 +216,7 @@
+ ADD_SUBDIRECTORY( xmpsdk )
+ ADD_SUBDIRECTORY( src )
+ 
+-IF( EXIV2_ENABLE_BUILD_SAMPLES )
+-    ADD_SUBDIRECTORY( samples )
+-ENDIF( EXIV2_ENABLE_BUILD_SAMPLES )
++ADD_SUBDIRECTORY( samples )
```

Comment:
isn't it easier to always enable EXIV2_ENABLE_BUILD_SAMPLES through cmake invocation instead of patching it here?